Web/Scotty/Internal/Types.hs

```diff
@@ -156,7 +156,7 @@ type Param = (Text, Text)
 type File = (Text, FileInfo LBS8.ByteString)
 
 data ActionEnv = Env { envReq       :: Request
-                     , envCaptureParams :: [Param]
+                     , envPathParams :: [Param]
```

Comment:
This is a breaking change (since the Types module is exposed) and should be marked as such.